aries-site/src/pages/design-tokens/global.mdx

```diff
@@ -0,0 +1,42 @@
+import { DesignTokensTable } from '../../components';
+import { ContentPane } from '../../layouts';
+
+## fontStack
+
+Font stack design tokens are a list of fonts that are used in the design system. The first font in the list is the primary font, and the rest are fallback fonts.
```

Comment:
Per the discussion with Andy from last Friday, should we consider mentioning the delivery mechanism of the fonts here? CDN, Brand Central for offline?

Should we cross reference to the "Foundations/Typography" and modify the content there?

Reply:
Yeah I think cross-linking for now is good. That ties in with @jcfilben comment here https://github.com/grommet/hpe-design-system/pull/4635#pullrequestreview-2610114399

Reply:
Done.